### ISSUE TITLE
ISO support for Blu-ray

### ISSUE THREAD

**Issue by Blurayx** [newloran2/emplexer#25]
Natively Dune HD players supports ISO Blu-ray, is it possible with emplexer.

**Comment by newloran2** [comment]
Supported added, but not tested ( i don't have any blu-ray iso ), plz test and report if work.

**Comment by Blurayx** [comment]
Hi newloran2

Thanks for the new update!

My ISO Blu-ray The Fifth Element DTS-HD Master plays flawlessly   

I did have one issue when I first installed emplexer, I was getting an error "format not supported", until I saved NFS again. After that ISO Blu-ray worked.
Not a real issue but I've attached a emplexer log file on plex forum if you want to have a look.
![screenshot4 2](https://f.cloud.github.com/assets/2873112/121162/89ce7676-6d78-11e2-9492-c45ce4fc3056.jpg)
